Packs/CVESearch/.pack-ignore

```diff
@@ -11,4 +11,13 @@ ignore=BA101
 ignore=RM104
 
 [file:CVESearchV2.yml]
-ignore=IN154
\ No newline at end of file
+ignore=IN154
+
+[file:CveReputation.yml]
+ignore=CJ105
```

Comment:
Why do we need this ignore?

Reply:
Don't even ask, because of build failures it was one of the attempts to make it work.